packages/web/src/dev/ValidatorsList.tsx

```diff
@@ -222,6 +245,12 @@ class ValidatorsListApp extends React.PureComponent<ValidatorsListProps & I18nPr
                 name="Uptime"
                 order={orderBy === 'uptime' ? orderAsc : null}
               />
+              <HeaderCell
+                onClick={this.orderBy.bind(this, 'attestation')}
```

Comment:
btw binding on every render will hurt reacts optimization heuristics as a new function will be created each render

Reply:
Should I create N functions for that? or there is a way more simple?

Reply:
maybe could do something in the constructor like 
```
Object.keys(orderByTypes).forEach(orderType => {
   this[`orderBy${orderType}`] = () => this.orderBy(orderType)
)
```

at least i think it would work.

Reply:
@Pedro-vk